### PR TITLE
Pod disruption budget for coredns

### PR DESCRIPTION
**Pull request by tomahawk28**
Toward https://github.com/giantswarm/giantswarm/issues/9054

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -146,6 +152,8 @@ data:
 
 - Remove `proxy` configuration support as it is [deprecated by upstream](https://coredns.io/2019/03/03/coredns-1.4.0-release/). New server block with `forward` plugin has to be used, more info in our [docs](https://docs.giantswarm.io/guides/advanced-coredns-configuration/).
 
+[v1.1.6]: https://github.com/giantswarm/coredns-app/pull/17
+[v1.1.5]: https://github.com/giantswarm/coredns-app/pull/16
```
Comment (tomahawk28) [inline CHANGELOG.md]: Was not there, added by now.

**Comment by tomahawk28** [comment]
Tested on ginger. 
```
---
# Source: coredns-app/templates/pdb.yaml
apiVersion: policy/v1beta1
kind: PodDisruptionBudget
metadata:
  name: coredns
  namespace: kube-system
spec:
  maxUnavailable: 2
  selector:
    matchLabels:
      k8s-app: coredns
---
```

File: helm/coredns-app/templates/pdb.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{{- if (gt (.Values.maxReplicas | int) 1) }}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ .Values.name }}
+  namespace: {{ .Values.namespace }}
```
Comment (rossf7) [inline helm/coredns-app/templates/pdb.yaml]: Do we need labels here?

Seems like some templates have them but others don't.

Comment (tomahawk28) [reply helm/coredns-app/templates/pdb.yaml]: The namespace? I think it's okay.

Comment (pipo02mix) [reply helm/coredns-app/templates/pdb.yaml]: I think Ross meant gs labels as other resources have

Comment (tomahawk28) [reply helm/coredns-app/templates/pdb.yaml]: 3 other GS labels from below,
```
  labels:
    giantswarm.io/service-type: "{{ .Values.serviceType }}"
    k8s-app: {{ .Values.name }}
    kubernetes.io/name: "CoreDNS"
```

Not sure we need those values for `pdb`.

Comment (tomahawk28) [reply helm/coredns-app/templates/pdb.yaml]: I will hold to release this until we make it sure not needed, sorry.

Comment (rossf7) [reply helm/coredns-app/templates/pdb.yaml]: I think all the resources should have these labels and that's what we say in `fmt`.

https://github.com/giantswarm/fmt/blob/master/kubernetes/annotations_and_labels.md#common-labels

Ji, can you add them and then create the release?

Comment (tomahawk28) [reply helm/coredns-app/templates/pdb.yaml]: Here the PR https://github.com/giantswarm/coredns-app/pull/18

**Pull request merged**